File: responses/T6CZBD-HJB/sum_of_integers_in_a_string.py
# you can write to stdout for debugging purposes, e.g.
# print("this is a debug message")

def solution(S):
    elements = S.split(',')
    output = 0
    for element in elements:
        if len(element) >= 2  and element[0] == '+':
            output += int(element[1:])
        elif len(element) >= 2 and element[0] == '-':
            output -= int(element[1:])
        elif len(element) == 1:
            output += int(element)
        
    return output

File: responses/SDXU35-KKU/sum_of_integers_in_a_string.js
// you can write to stdout for debugging purposes, e.g.
// console.log('this is a debug message');

const sum = (val, next) => val + next;

function solution(S) {
    let numbers = [];
    let numStrings = S.split(',');
    
    for (number of numStrings) {
        numbers.push(parseInt(number, 10));
    }
    
    return numbers.reduce(sum);
}